src/api/apps/articles/model/save.coffee

```diff
@@ -25,7 +25,7 @@ chalk = require 'chalk'
 
 @onUnpublish = (article, cb) =>
   @generateSlugs article, (err, article) =>
-    deleteArticleFromSailthru getArticleUrl(article), =>
+    getArticleUrl(article), =>
```

Comment:
```suggestion
    getArticleUrl(article) =>
```

This leftover comma seems to be [breaking the test job in CI](https://app.circleci.com/pipelines/github/artsy/positron/4791/workflows/f9c01959-7872-47a3-8d7e-b6f7bf5b2fb5/jobs/10351?invite=true#step-104-253).